Automated Smoketest Build: failed
Number of Error(s): 5
/home/ati/smoketest/smoketest-8021/HPCC-Platform/docs/HPCCCertify/Cert-Mods/CertPreflight.xml:298: element sect2: validity error : ID Expected_Results_Data_refinery already defined
/home/ati/smoketest/smoketest-8021/HPCC-Platform/docs/HPCCCertify/Cert-Mods/CertPreflight.xml:298: element sect2: validity error : ID Expected_Results_Data_refinery already defined
make[2]: *** [Release/docs/HPCCCertification-6.0.0-beta1.pdf] Error 1
make[1]: *** [docs/HPCCCertify/CMakeFiles/HPCCCertification-6.0.0-beta1.pdf.dir/all] Error 2
make: *** [all] Error 2

--- a/docs/ECLLanguageReference/ECLR_mods/BltInFunc-SEQUENTIAL.xml
+++ b/docs/ECLLanguageReference/ECLR_mods/BltInFunc-SEQUENTIAL.xml
@@ -40,10 +40,7 @@
   <para>The <emphasis role="bold">SEQUENTIAL </emphasis>action executes the
   items in the <emphasis>actionlist</emphasis> in the order in which they
   appear in the <emphasis>actionlist</emphasis>. This is useful when a
-  subsequent action requires the output of a precedent action. By definition,
-  PERSIST on an attribute means the attribute is evaluated outside of any
-  given evaluation order. Therefore, SEQUENTIAL has no effect on PERSISTed
-  attributes.</para>
+  subsequent action requires the output of a precedent action.</para>
 
   <para>Example:</para>
 
@@ -61,6 +58,5 @@ SEQUENTIAL(Act1,PARALLEL(Act2,Act3));
           Act2 and Act3 together
 </programlisting>
 
-  <para>See Also: <link linkend="PARALLEL_function">PARALLEL</link>, <link
-  linkend="PERSIST">PERSIST</link></para>
+  <para>See Also: <link linkend="PARALLEL_function">PARALLEL</link></para>
 </sect1>

--- a/docs/ECLLanguageReference/ECLR_mods/WkFlo-PERSIST.xml
+++ b/docs/ECLLanguageReference/ECLR_mods/WkFlo-PERSIST.xml
@@ -153,10 +153,6 @@
   <emphasis>attribute</emphasis> is used in the computation. This gives a kind
   of "compute in anticipation" capability.</para>
 
-  <para>By definition, PERSIST on an attribute means the attribute is
-  evaluated outside of any given evaluation order. Therefore, SEQUENTIAL has
-  no effect on PERSISTed attributes.</para>
-
   <para>Example:</para>
 
   <programlisting>  CountPeople := COUNT(Person) : PERSIST('PeopleCount');

